starknet_py/net/account/_account_proxy.py

```diff
@@ -36,6 +42,16 @@ async def get_balance(
     ) -> int:
         return await self._account_client.get_balance(token_address=token_address)
 
+    async def sign_for_fee_estimate(
+        self, transaction: TypeAccountTransaction
+    ) -> TypeAccountTransaction:
+        version = self.supported_transaction_version + 2**128
+        signature = self._account_client.signer.sign_transaction(transaction)
```

Comment:
Version should be added to the transaction before calculating a signature

Reply:
Fixed